sklearn/model_selection/_split.py

```diff
@@ -2072,7 +2072,12 @@ def check_cv(cv=5, y=None, *, classifier=False):
     return cv  # New style cv objects are passed without any modification
 
 
-def train_test_split(*arrays, **options):
+def train_test_split(*arrays,
+                     test_size=None,
```

Comment:
Actually let's make these key word only with
```py
def train_test_split(*arrays,
                     *,
                     test_size=None,
                     ...
                     )
```
maybe the previous approach was a way to force them to be key word only with Python 2.

Reply:
Thanks for your feedback! I actually learned something new :)